components/automate-ui/src/app/components/pending-edits-bar/pending-edits-bar.component.html

```diff
@@ -1,8 +1,15 @@
 <div id="pending-edits-bar" [hidden]="isBarHidden">
-  <span class="info">Project edits pending, update <a [routerLink]="['/settings/projects']">projects</a> to apply changes.</span>
-  <app-authorized [allOf]="['/iam/v2/apply-rules', 'post']">
-    <button mat-button (click)="openConfirmUpdateStartModal()">Update Projects</button>
-  </app-authorized>
+  <div [className]="updateProjectsFailed ? 'failure-container' : 'normal-container'">
+    <span *ngIf="!updateProjectsFailed" class="info">
+      Project edits pending, update <a [routerLink]="['/settings/projects']">projects</a> to apply changes.
+    </span>
+    <span *ngIf="updateProjectsFailed" class="info">
+      Project update failed. Review system status from the command line with <pre>chef-automate status</pre> and try again.
+    </span>
```

Comment:
Unlike the separation of pending edits vs. update in progress, that really have different functionality, the distinction between pending edits and pending-edits-due-to-failed-update is really the same functionality. That is why I just made this component handle both variations rather than creating a new component (which would have addded significant overhead for its infrastructure in layout facade).
In fact, here we already have the variable we need to distinguish--this.updateProjectsFailed.